xAPI-Data.md

```diff
@@ -2201,6 +2202,7 @@ Durations are strings representing the amount of time something took.
 * A Duration MUST be expressed using the format for duration in ISO 8601:2004(E) section 4.4.3.2.
 The alternative format (in conformity with the format used for time points and described in ISO 8601:2004(E) 
 section 4.4.3.3) MUST NOT be used.
+* A Duration SHOULD* be expressed using the format described in [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt), which is a profile of ISO 8601. 
```

Comment:
This confused me when I first looked at RFC 3339: the introduction states, explicitly, that "Description of time periods, or intervals, is not covered here." The only coverage of durations is in the appendix, where they make a general ISO8601 ABNF grammar. So, I think it might be helpful to specify "Appendix A" here.